public/_data.json

```diff
@@ -1,35 +1,18 @@
 {
     "current": {
-        "title": "",
-        "datetime": "Saturday December 9th, 2017 5pm",
-        "date": "12/18/2017 17:00:00",
-        "host": {
-            "address": "9400 Culver Blvd, Culver City, CA 90232",
-            "name": "Culver Hotel",
-            "misc": "",
-            "url": "http://www.culverhotel.com/"
-        },
-        "drinksjs": {
-            "address": null,
-            "name": null,
-            "misc": null,
-            "url": null
-        },
-        "speakers": [
```

Comment:
is it possible to just set these values to null instead of deleting them? I know I would totally forget what they are and hate having to dig into github to find them again.

Reply:
yeah totally, I was thinking that as well.